lib/src/icons/Carat-Down.tsx

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+
+import './Carat-Down.scss';
+
+const CaratDown = () =>
+  <svg viewBox="0 0 12 8" className="nx-icon nx-icon--carat-down">
+    <path fill="currentColor"
+          fill-rule="nonzero"
```

Comment:
This is causing a console error because it should be `fillRule` not `fill-rule`.  Also I doubt it's necessary, try removing it

Reply:
> This is causing a console error because it should be `fillRule` not `fill-rule`. Also I doubt it's necessary, try removing it

https://github.com/sonatype/sonatype-react-shared-components/pull/55/commits/01f6164604ec672486d935eb13c9eb4c3e460364